src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-zonegroup-form/rgw-multisite-zonegroup-form.component.html

```diff
@@ -198,7 +198,7 @@
     <div class="modal-footer">
       <cd-form-button-panel (submitActionEvent)="submit()"
                             [form]="multisiteZonegroupForm"
-                            [submitText]="(action | titlecase) + ' ' + (resource | upperFirst)"></cd-form-button-panel>
+                            [submitText]="(action | titlecase) + ' ' + 'Zone Group'"></cd-form-button-panel>
```

Comment:
what happened to the `resource`? Can't we change the resource to `zone group` and use [titleCase](https://angular.io/api/common/TitleCasePipe) pipe here instead of `upperFirst`?